src/TMxpMudlet.cpp

```diff
@@ -101,13 +101,13 @@ TMxpTagHandlerResult TMxpMudlet::tagHandled(MxpTag* tag, TMxpTagHandlerResult re
 
 void TMxpMudlet::enqueueMxpEvent(MxpStartTag* tag)
 {
-    TMxpEvent ev;
-    ev.name = tag->getName();
+    TMxpEvent event;
```

Comment:
In some other places in the code-base we end up manipulating both Qt and Mudlet type events and in that case I've resorted to calling the latter something like `mudletEvent` to destinguish it from the former. It isn't obvious (I hadn't noticed before) that we now seem to have a `TMxpEvent` class, so perhaps it would help to use:
```suggestion
    TMxpEvent mxpEvent;
```
here instead...?

You may wish to action this.

Reply:
Applied.